src/test/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilderTest.java

```diff
@@ -7,42 +7,70 @@
 import bio.terra.common.CloudPlatformWrapper;
 import bio.terra.common.category.Unit;
 import bio.terra.model.CloudPlatform;
+import bio.terra.model.SnapshotBuilderDomainOption;
 import bio.terra.service.snapshotbuilder.query.TablePointer;
 import bio.terra.service.snapshotbuilder.query.TableVariable;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.EnumSource;
 import org.mockito.junit.jupiter.MockitoExtension;
+import org.testcontainers.shaded.org.apache.commons.lang3.NotImplementedException;
 
 @ExtendWith(MockitoExtension.class)
 @Tag(Unit.TAG)
 class SearchConceptsQueryBuilderTest {
+
+  private SnapshotBuilderDomainOption createTestSnapshotBuilderDomainOption(
+      String category, int id) {
+    SnapshotBuilderDomainOption domainOption = new SnapshotBuilderDomainOption();
+    domainOption.setCategory(category);
+    domainOption.setId(id);
+    return domainOption;
+  }
+
+  String formatSQLWithLimit(String sql, CloudPlatformWrapper cloudPlatformWrapper) {
+    int limit = 100;
+    if (cloudPlatformWrapper.isAzure()) {
+      return String.format("TOP %d %s", limit, sql);
```

Comment:
the sql string should start with SELECT then TOP n. But also is this used anywhere?

Reply:
I ended up deleting this